examples/platform/silabs/efr32/rs911x/hal/efx_spi.c

```diff
@@ -343,9 +322,17 @@ sl_status_t sl_wfx_host_pre_lcd_spi_transfer(void)
 
 sl_status_t sl_wfx_host_post_lcd_spi_transfer(void)
 {
+    uint32_t status = SL_STATUS_OK;
     USART_Enable(SL_MEMLCD_SPI_PERIPHERAL, usartDisable);
     CMU_ClockEnable(SPI_CLOCK(SL_MEMLCD_SPI_PERIPHERAL_NO), false);
     GPIO->USARTROUTE[SL_MEMLCD_SPI_PERIPHERAL_NO].ROUTEEN = PINOUT_CLEAR;
+    status = sl_board_disable_display();
+    if (SL_STATUS_OK != status)
+    {
+        SILABS_LOG("sl_board_disable_display failed with error: %x", status);
+        xSemaphoreGive(spi_sem_sync_hdl);
```

Comment:
why is this xSemaphoreGive not wrapped by `#if SL_SPICTRL_MUX` but the one at line 337 is?
They should be the same. If they are indeed the same why duplicate the give and return?

simply doing the following does the same more cleanly.
```
if (SL_STATUS_OK != status)
{
    SILABS_LOG("sl_board_disable_display failed with error: %x", status);
}

xSemaphoreGive(spi_sem_sync_hdl);
return status;
```